### ISSUE TITLE
Remove site-specific setup configurations from origin

### ISSUE THREAD

**Issue by Ejler** [CINF/cinfdata#8]
Like we are separating PyExpLabSys/machines into its own git archive, so we should do with cinfdata to make it more shareable. Probably by repeating the solution of putting it in the same folder as cinfdata, which would be in /var/www or /var/www/html? This might require some extra security settings to allow/deny access to (certain) files.

**Comment by robertjensen** [comment]
Agreed. A solution very similar to the great success of the individual `machines` repositories for PyExpLabSys would be great. What would be a suitable name?
 * setups (maybe a bit too generic?)
 * cinfdata_setups (easy to understand, but not entirely symmetric with machines)
 * chambers (would fit the tradition, but many items really are not chambers)
 * ....
